### DISCUSSION TITLE
No worky

### DISCUSSION THREAD

**Discussion by yuzu1** [vrcx-team/VRCX#493] [Q&A] [answered]
_"Enter a numeric code that was sent to your email."_

*Enters code*

_429 Too Many Requests
"Too many requests"
"auth/twofactorauth/emailotp/verify"_

???

**Comment by Myrkie** [comment]
Update your vrcx to latest version and try again

**Reply by yuzu1** [reply]
I'm already using the latest. I updated it right after I got the error on the previous version I was using.

**Comment by yuzu1** [comment, marked as answer]
It just decided to work now, problem solved. :)

**Comment by Natsumi-sama** [comment]
VRChat rate limit moment

**Reply by Myrkie** [reply]
xd
